index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"/>
+    <title>Grow Megha Portfolio</title>
+  </head>
+
+  <body>
+    <header class="headercontainer">
+      <div class="logo">Megha</div>
+          <div><i class="fa-solid fa-bars hambrgricon"></i></div>
```

Comment:
- [ ] I suggest using HTML5 semantic tags (e.g., header, nav) over divs to Follow [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), Great job using `<header>` tag here 💯, However I suggest using `<nav>` tag for the hamburger button instead of  `<div>` 👌
```
<nav><i class="fa-solid fa-bars hambrgricon"></i></nav>
```

Reply:
- [ ] [REQUIRED] It will be important to follow what the previous reviewer suggested, please fix it to make your project follow best practices. 


> * [ ]  I suggest using HTML5 semantic tags (e.g., header, nav) over divs to Follow [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), Great job using `<header>` tag here 100, However I suggest using `<nav>` tag for the hamburger button instead of  `<div>` 
> 
> ```
> <nav><i class="fa-solid fa-bars hambrgricon"></i></nav>
> ```